st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -113,6 +113,7 @@ def __init__(self, hostname, port=22, username=None, password=None, bastion_host
         self.bastion_host = bastion_host
         self.bastion_client = None
         self.bastion_socket = None
+        self.passphrase = passphrase
```

Comment:
I think it's a good idea to throw here if `passphrase` is provided, but private key is not.